src/mjolnir/landmark_database_builder.cc

```diff
@@ -164,7 +148,7 @@ std::vector<Landmark> LandmarkDatabase::get_landmarks_in_bounding_box(const doub
   sqlite3_bind_double(bounding_box_stmt, 3, maxLong);
   sqlite3_bind_double(bounding_box_stmt, 4, maxLat);
 
-  LOG_INFO(sqlite3_expanded_sql(bounding_box_stmt));
+  LOG_TRACE(sqlite3_expanded_sql(bounding_box_stmt));
```

Comment:
i pretty much removed all logs except:

1. the destructor logs when vacuum analyze fails
2. insert/select log trace now because anything else is too verbose